test/tools/cluster_setup.sh

```diff
@@ -10,19 +10,38 @@ DATA_DIR=${DATA_DIR:-data}
 SINGLE_DIR=${SINGLE_DIR:-$DATA_DIR/server}
 REPLICASET_DIR=${REPLICASET_DIR:-$DATA_DIR/replica_set}
 SHARDED_DIR=${SHARDED_DIR:-$DATA_DIR/sharded_cluster}
+AUTH=${AUTH:-auth}
 
-if [[ $1 == "replica_set" ]]; then
-    mkdir -p $REPLICASET_DIR
-    mlaunch init --dir $REPLICASET_DIR --replicaset --nodes 3 --arbiter --name rs --port 31000 --enableMajorityReadConcern --setParameter enableTestCommands=1
-    echo "mongodb://localhost:31000,localhost:31001,localhost:31002/?replicaSet=rs"
-elif [[ $1 == "sharded_cluster" ]]; then
-    mkdir -p $SHARDED_DIR
-    mlaunch init --dir $SHARDED_DIR --replicaset --nodes 3 --arbiter --name rs --port 51000 --enableMajorityReadConcern --setParameter enableTestCommands=1 --sharded 1 --mongos 2
-    echo "mongodb://localhost:51000,localhost:51001"
-elif [[ $1 == "server" ]]; then
-    mkdir -p $SINGLE_DIR
-    mlaunch init --dir $SINGLE_DIR --single --setParameter enableTestCommands=1
-    echo "mongodb://localhost:27017"
+if [[ "$AUTH" == "auth" ]]; then
+    if [[ $1 == "replica_set" ]]; then
```

Comment:
I would be okay with the script always turning on auth unconditionally, if I need to go out of my way to disable it then writing out the command is fine. I'm thinking keeping this wrapper script simple is ideal, and it keeps us all mostly on the same page with local testing. thoughts?

Reply:
Yup, I think that's reasonable!